app/views/shared/_tax_return_list.html.erb

```diff
@@ -58,7 +58,7 @@
       <% end %>
     </li>
   <% end %>
-  <% if tax_return_creatable && client.tax_returns.length != TaxReturn.filing_years.length %>
+  <% if tax_return_creatable && client.tax_returns.pluck(:year).sort != MultiTenantService.new(:gyr).filing_years.sort %>
```

Comment:
this logic seems a little different: instead of checking array length we're checking that the arrays have the same contents?

Reply:
agreed